packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/loggingservice/LoggingResource.java

```diff
@@ -14,22 +17,24 @@
 import javax.ws.rs.Produces;
 
 @Path("/api/")
```

Comment:
I imagine that this is testing and works, but I noticed that the other classes omit the last / and just have `@Path("/api")`

Reply:
yeah definitely don't need that :P